api/apps/api/test/scenario-solutions/get-all-solutions.fixtures.ts

```diff
@@ -0,0 +1,118 @@
+import { GivenUserExists } from '../steps/given-user-exists';
+import { UsersScenariosApiEntity } from '@marxan-api/modules/access-control/scenarios-acl/entity/users-scenarios.api.entity';
+
+export const getFixtures = async () => {
+  const app = await bootstrapApplication();
+  const ownerToken = await GivenUserIsLoggedIn(app, 'aa');
+  const contributorToken = await GivenUserIsLoggedIn(app, 'bb');
+  const viewerToken = await GivenUserIsLoggedIn(app, 'cc');
+  const contributorUserId = await GivenUserExists(app, 'bb');
+  const viewerUserId = await GivenUserExists(app, 'cc');
+  const scenarioViewerRole = ScenarioRoles.scenario_viewer;
+  const scenarioContributorRole = ScenarioRoles.scenario_contributor;
+  const { projectId, cleanup: cleanupProject } = await GivenProjectExists(
+    app,
+    ownerToken,
+    {
+      countryCode: 'BWA',
+      adminAreaLevel1Id: 'BWA.12_1',
+      adminAreaLevel2Id: 'BWA.12.1_1',
+    },
+  );
+  const scenario = await ScenariosTestUtils.createScenario(app, ownerToken, {
+    ...E2E_CONFIG.scenarios.valid.minimal(),
+    projectId,
+  });
+  const scenarioId = scenario.data.id;
+  const marxanOutputRepo: Repository<ScenariosOutputResultsApiEntity> = app.get(
+    getRepositoryToken(ScenariosOutputResultsApiEntity),
+  );
+
+  const userScenariosRepo: Repository<UsersScenariosApiEntity> = app.get(
+    getRepositoryToken(UsersScenariosApiEntity),
+  );
+
+  return {
+    GivenScenarioHasSolutionsReady: async () => {
+      await marxanOutputRepo.save(
+        marxanOutputRepo.create({
+          scenarioId,
+          runId: 1,
+          scoreValue: 4000,
+          costValue: 2000,
+          missingValues: 1,
+          planningUnits: 123,
+        }),
+      );
+    },
+    GivenContributorWasAddedToScenario: async () =>
+      await userScenariosRepo.save({
+        scenarioId: scenarioId,
+        roleName: scenarioContributorRole,
+        userId: contributorUserId,
+      }),
+
+    GivenViewerWasAddedToScenario: async () =>
+      await userScenariosRepo.save({
+        scenarioId: scenarioId,
+        roleName: scenarioViewerRole,
+        userId: viewerUserId,
+      }),
+    WhenGettingSolutionsAsOwner: async () =>
+      request(app.getHttpServer())
+        .get(`/api/v1/scenarios/${scenarioId}/marxan/solutions`)
+        .set('Authorization', `Bearer ${ownerToken}`),
+    WhenGettingSolutionsAsContributor: async () =>
+      request(app.getHttpServer())
+        .get(`/api/v1/scenarios/${scenarioId}/marxan/solutions`)
+        .set('Authorization', `Bearer ${contributorToken}`),
+    WhenGettingSolutionsAsViewer: async () =>
+      request(app.getHttpServer())
+        .get(`/api/v1/scenarios/${scenarioId}/marxan/solutions`)
+        .set('Authorization', `Bearer ${viewerToken}`),
+    ThenSolutionsShouldBeResolved: async (response: request.Response) => {
+      expect(response.body.meta).toMatchInlineSnapshot(`
```

Comment:
Again, I prefer using `.toEqual()`, but above all I prefer that we use the same method (any of them) everywhere
```suggestion
      expect(response.body.meta).toEqual({
        "page": 1,
        "size": 25,
        "totalItems": 1,
        "totalPages": 1,
      })
```